packages/core/src/services/ai/index.ts

```diff
@@ -95,10 +98,10 @@ export async function ai({
     const result = await streamObject({
       ...commonOptions,
       schema: jsonSchema(schema),
-      // @ts-expect-error - output is valid but depending on the type of schema
+      // output is valid but depending on the type of schema
       // there might be a mismatch (e.g you pass an object schema but the
       // output is "array"). Not really an issue we need to defend atm.
-      output,
+      output: output as any,
       onFinish,
     })
```

Comment:
For some reason building this now fails with the following error message:
`../../core/src/services/ai/index.ts(101,7): error TS2578: Unused '@ts-expect-error' directive.`
But removing it introduces a type error 😭

Reply:
just add the ts-expect-error at the bottom of the comment instead of the top (right before the output argument)